### PR TITLE
Add integration test for OIDC federated identity provider initiated logout

### PR DESCRIPTION
**Pull request by ThaminduR**
Add integration test for OIDC federated identity provider initiated logout

Related to  - https://github.com/wso2/product-is/issues/9254

### CODE REVIEW

File: modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oidc/OIDCFederatedIdpInitLogoutTest.java [hunk 1]
```diff
@@ -0,0 +1,875 @@
+/*
+ * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (mefarazath) [inline modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oidc/OIDCFederatedIdpInitLogoutTest.java:2]:
```suggestion
 * Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

File: modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oidc/OIDCFederatedIdpInitLogoutTest.java [hunk 2]
```diff
@@ -0,0 +1,875 @@
...
+    private static final int PORT_OFFSET_0 = 0;
+    private static final int PORT_OFFSET_1 = 1;
+
+    CookieStore cookieStore;
+    private CloseableHttpClient client;
+
+    @DataProvider(name = "configProvider")
+    public static Object[][] configProvider() {
+
+        return new Object[][]{{TestUserMode.SUPER_TENANT_ADMIN}};
```
Comment (mefarazath) [inline modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oidc/OIDCFederatedIdpInitLogoutTest.java:166]: Can we add the tenant admin here as well?

File: modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oidc/OIDCFederatedIdpInitLogoutTest.java [hunk 3]
```diff
@@ -0,0 +1,875 @@
...
+        Assert.assertTrue(userCreated, "User creation failed in secondary IS.");
+    }
+
+    @AfterClass(alwaysRun = true)
+    public void endTest() throws Exception {
+
+        try {
+            super.deleteServiceProvider(PORT_OFFSET_0, PRIMARY_IS_SP_NAME);
+            super.deleteIdentityProvider(PORT_OFFSET_0, PRIMARY_IS_IDP_NAME);
+
```
Comment (mefarazath) [inline modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oidc/OIDCFederatedIdpInitLogoutTest.java:217]:
```suggestion
```

**Comment by ThaminduR** [comment]
New PR - https://github.com/wso2/product-is/pull/11579

**Pull request closed without merging**